CHANGELOG.markdown

```diff
@@ -5,6 +5,7 @@ HEAD
 * Add `dimapTraversing`, `lmapTraversing`, and `rmapTraversing` to `Data.Profunctor.Traversing`
 * Add documentation stating the laws for various profunctors.
 * Introduce the `Data.Profunctor.Yoneda` module.
+* Mark `Data.Profunctor.Types` trustworthy. Note that the module effectively reexports`Data.Coerce.coerce`, overriding its unsafe status in `base`.
```

Comment:
Good question, my memory tricked me here. The `profunctors` package does effectively reexport `Data.Coerce.coerce` but the reason is that `Data.Profunctor.Unsafe` is marked `Trustworthy`, and that's already the case in version 5.2. (It's not the only package doing that; `lens` has `Control.Lens.Iso.coerced`, for example)

Since there's no new pitfall here, the added `Trustworthy` is a minor change. I guess we don't need this changelog entry at all.